Clarify how to go about assessing or trialing a new technology

mwz opened the pull request.

brad-allan-ovo commented on README.md:
```diff
@@ -1,9 +1,12 @@
 # OVO.com Technology Radar
 
 The OVO.com Technology Radar is a living document that attempts to capture and
```
```suggestion
The OVO.com Technology Radar is a living document that captures and
```

brad-allan-ovo commented on README.md:
```diff
@@ -1,9 +1,12 @@
-give a high-level overview of our technology landscape. It's also a tool that
-helps us in discussions about various technologies that we want to use and
-invest in as an organisation.
+give a high-level overview of our technology landscape.
```
```suggestion
provides a high-level overview of our technology landscape.
```

brad-allan-ovo reviewed (commented):
I would prefer that technology governance and the introduction of it (effectively) is shared in a live forum where Q&A can happen and in the end we as a tech org commit to the governance.

lsale reviewed (changes requested):
I'd recommend we slightly modify the README for simplicity so we can highlight certain sections. For example, I'd add the following:
*Why should you use the radar?* - you cover it but it's in a paragraph. I'd make it a different section

*Why do I see adopt only* - I don't think we cover that

*What to do if my technology is not listed here* - I'd make this a different section

*What to do if my technology is listed in adopt* - I think we need a bit of guidance around this. Whilst adopt means "go ahead, choose it" we also need to explain that it's good practice to talk to other teams using that technology to understand how they use it. So design/architect first by talking to other teams

mwz commented:
Hi @lsale, I've addressed your feedback - I've captured those points in a FAQ section because they were all questions, but let me know if you'd prefer to have those as standalone sections in the readme.

lsale reviewed (approved):
Looks great!
